pkgs/applications/networking/nextcloud-client/default.nix

```
@@ -20,29 +20,15 @@ mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig cmake ];
 
-  buildInputs = [ qtbase qtwebkit qtkeychain qttools qtwebengine sqlite openssl.out pcre inotify-tools libcloudproviders ];
-
-  enableParallelBuilding = true;
-
-  NIX_LDFLAGS = "${openssl.out}/lib/libssl.so ${openssl.out}/lib/libcrypto.so";
-
-  cmakeFlags = [
-    "-UCMAKE_INSTALL_LIBDIR"
-    "-DCMAKE_BUILD_TYPE=Release"
-    "-DOPENSSL_LIBRARIES=${openssl.out}/lib"
-    "-DOPENSSL_INCLUDE_DIR=${openssl.dev}/include"
-    "-DINOTIFY_LIBRARY=${inotify-tools}/lib/libinotifytools.so"
-    "-DINOTIFY_INCLUDE_DIR=${inotify-tools}/include"
-  ];
+  buildInputs = [ qtbase qtwebkit qtkeychain qttools qtwebengine sqlite openssl pcre inotify-tools libcloudproviders ];
 
   qtWrapperArgs = [
     ''--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libsecret ]}''
```

```suggestion
    "--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libsecret ]}"
```

Funny when you author one case that does this and somehow it's used everywhere :smile:

---

Yeah, it is contagious. I myself was infected by this fad in placeholders :mask:

---

Right, perhaps when there's actually tooling for the code style it can be a CI check.